fang/src/asynk/async_worker.rs

```diff
@@ -539,4 +567,33 @@ mod async_worker_tests {
     async fn insert_task(test: &mut AsyncQueue<NoTls>, task: &dyn AsyncRunnable) -> Task {
         test.insert_task(task).await.unwrap()
     }
+
+    #[tokio::test()]
```

Comment:
I would not use those parentheses in `tokio::test` for consistency with the rest of the project, but other than that nitpick, this PR seems good.

Reply:
[delete parenthesis](https://github.com/ayrat555/fang/pull/136/commits/fa6ee4da1286fb0fe1f361e8fd707f60a535f068)